access/types/stdasync.py

```diff
@@ -433,16 +433,6 @@ def _acceptSubmission(request, course, exercise, post_url, sdir):
             r["code"], r["out"], r["err"])
         qlen = 1
 
-    # Queue in celery & rabbitmq for chroot sandbox actions.
-    else:
```

Comment:
The `container_flag` can still be false, which would silently ignore the submission, so that variable should be removed (also the settings value and so on).

Reply:
The logic here is a bit abstract. Nevertheless, I will modify the lines where the `container_flag` is included. Please, review the changes in the next pull request.